dev/text-field.html

```diff
@@ -4,18 +4,89 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Text field</title>
+    <title>Text Field</title>
     <script type="module" src="./common.js"></script>
 
     <script type="module">
-      import '@vaadin/text-field';
-      import '@vaadin/tooltip';
+      import '@vaadin/text-field/src/vaadin-lit-text-field.js';
+      import '@vaadin/icon/src/vaadin-lit-icon.js';
+      import '@vaadin/vaadin-lumo-styles/vaadin-iconset.js';
+      import '@vaadin/icons';
     </script>
   </head>
 
   <body>
-    <vaadin-text-field label="User name" required>
-      <vaadin-tooltip slot="tooltip" text="Other users will see this"></vaadin-tooltip>
-    </vaadin-text-field>
+    <section>
+      <h2>Plain</h2>
+      <vaadin-text-field value="Value"></vaadin-text-field>
+      <vaadin-text-field placeholder="Placeholder"></vaadin-text-field>
+    </section>
+
+    <section>
+      <h2>Bells & Whistles</h2>
+      <vaadin-text-field
+          label="Label"
```

Comment:
Consider applying Prettier formatting to this file.

Reply:
But it makes it really hard to read 😅